Gh action fails

Example here: https://github.com/spine-generic/data-multi-subject/runs/1696448913

The issue seems to come from:

```
Run # install spine-generic for acquisition parameters check
Collecting spine-generic@ git+https://github.com/spine-generic/spine-generic.git@v2.3
  Cloning https://github.com/spine-generic/spine-generic.git (to revision v2.3) to /tmp/pip-install-1ry77eih/spine-generic_af89ee66787b4049895a9478174456b1
  WARNING: Generating metadata for package spine-generic produced metadata for project name spinegeneric. Fix your #egg=spine-generic fragments.
ERROR: Requested spinegeneric from git+https://github.com/spine-generic/spine-generic.git@v2.3 has different name in metadata: 'spinegeneric'
WARNING: You are using pip version 20.3.1; however, version 20.3.3 is available.
You should consider upgrading via the '/opt/hostedtoolcache/Python/3.7.9/x64/bin/python -m pip install --upgrade pip' command.
Error: Process completed with exit code 1.

```

--- a/.github/workflows/python-app.yml
+++ b/.github/workflows/python-app.yml
@@ -24,7 +24,7 @@ jobs:
     - name: before_install
       run: |
         # install spine-generic for acquisition parameters check
-        pip install spine-generic@git+https://github.com/spine-generic/spine-generic.git@v2.3
+        pip install spinegeneric@git+https://github.com/spine-generic/spine-generic.git@v2.3
 
         # use conda's git-annex, because it's the most up to date;
         # Debian's is very behind, and even neurodebian's is behind

